scripts/lighthouseRun.sh

```diff
@@ -4,3 +4,5 @@ npx -p lighthouse@11.3.0 lighthouse http://localhost:7080/news/articles/cn7k01xp
 npx -p lighthouse@11.3.0 lighthouse http://localhost:7080/hindi --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
 npx -p lighthouse@11.3.0 lighthouse http://localhost:7081/pidgin/live/c7p765ynk9qt?renderer_env=test --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
 npx -p lighthouse@11.3.0 lighthouse http://localhost:7081/pidgin/live/c07zr0zwjnnt?renderer_env=test --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
+npx -p lighthouse@11.3.0 lighthouse http://localhost:7081/mundo/send/u50853489?renderer_env=live --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
+npx -p lighthouse@11.3.0 lighthouse http://localhost:7081/somali/send/u130092370?renderer_env=test --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
```

Comment:
What's the difference between the 2 uploader forms here (and do we need to test both of them?)

Reply:
Thanks for explaining, makes sense